Makefile

```diff
@@ -19,6 +19,7 @@ b benchmark benchmarks:
 
 .PHONY: l ld leak leak-detection
 l ld leak leak-detection:
+	npm i memwatch-next --no-save
```

Comment:
Can we use `@airbnb/node-memwatch` instead here. It's compatible with Node 8/10. And then can you please update the `Leak Detection` section of the readme explaining that it only works on Node 8/10.